src/db/SearchClientBase.ts

```diff
@@ -0,0 +1,3 @@
+import { EventEmitter } from 'events';
+
+export class SearchClientBase extends EventEmitter { };
```

Comment:
```suggestion
export class SearchClientBase extends EventEmitter { }
```

CC: @tamaina this is a class so a semicolon isn't needed.

Reply:
Why?

Reply:
I'm verry sorry